controllers/drcluster_controller.go

```diff
@@ -705,6 +705,10 @@ func (u *drclusterInstance) fenceClusterOnCluster(peerCluster *ramen.DRCluster)
 		return true, fmt.Errorf("failed to get NetworkFence using MCV (error: %w)", err)
 	}
 
+	if nf.Spec.FenceState != csiaddonsv1alpha1.FenceState(ramen.ClusterFenceStateFenced) {
```

Comment:
I think what you wanted is:
```
if peerCluster.Spec.ClusterFence != nf.Spec.FenceState {
   return ...
}
...

Reply:
I have to match the nf resource field with a value in their API for fence.
I can update it to 
```
nf.Spec.FenceState != csiaddonsv1alpha1.FenceState(peerCluster.Spec.ClusterFence)
```